### DISCUSSION TITLE
OSM map background

### DISCUSSION THREAD

**Discussion by ASamSam** [eclipse-sirius/sirius-components#3619] [Q&A]
Hello,
I'm pretty sure that on one of your presentations, I saw the OSM map as a background for the view, but it was just 1 shot; I don't know the details.
Can you please tell me if it is possible to establish a view with nodes and edges on the OSM map? with lat/lng positions?
Sincerely yours, Andrew

**Comment by sbegaudeau** [comment]
Hi,

I think that it was in one of my presentations a couple years ago (it must be somewhere on Youtube):

<img width="2103" alt="Capture d’écran 2022-06-14 à 17 02 13" src="https://github.com/eclipse-sirius/sirius-web/assets/422587/74326966-097f-426e-b95e-096ca3b85e88">

 It was not a diagram, it was a new kind of representation dedicated to Google Maps to show that the list of representations in Sirius Web is not hardcoded and Sirius Web can be extended with additional representations. Something like that can be done quite easily with a React component used to display Google Maps.

Kind regards,

Stephane Begaudeau, Obeo

**Reply by ASamSam** [reply]
Hello. thank you very much for the answer! it means, that graphical positions became lat/lng.
When some Leablef-based tools are showing data on the map, they do care about loading only 'visible' part of objects. This also can be implemented with Sirius?
